contents/handbook/company/structure.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Small Teams
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+PostHog is structured for speed, autonomy and innovation.
+
+Traditional organizations have big, separate functions. You have a product team, an engineering team, customer support, and so on. This slows things down, which stifles innovation - you have to get your boss to talk to someone else's boss to get work done. It also means that people can't really see the impact of their work.
+
+PostHog started off as a completely flat company with one big goal: to increase the number of successful products in the world.
+
+As we are getting bigger, we realized that it's harder for people to see the direct impact of their work, which reduces the sense of ownership.
+
+We have therefore introduced Small Teams. These are designed to each operate like a startup. 
+
+# How it works
+
+* A Small Team should never be more than ten people
+* A Small Team has an Accountable Person responsible for its performance - whoever is most appropriate depending on what the team is working on. Who would make the right founder for that bit of the company?
+* A Small Team must have (1) a customer, (2) a mission and (3) metrics
+* There may be certain functions where at our current stage we don't need a Small Team yet.
+* Each small team runs its own retrospective + sprint every week for 30 mins.
+
+# Small Teams list
+
+* Core analytics
+* Feature flags and a/b testing
+* VPC deployment
+* Website and docs
+* Extensibility: plugins and APIs
+* Session recording
+* Event capture (SDKs, autocapture)
+* Growth engineering (customer = _internal_ teams above)
+
+# Functional teams
+
+* Ops
+* Marketing
+* Customer Success
+
+# FAQ
+
+## Who do Small Teams report to?
+
+Small teams _may_ be nested in future. In the short run, this isn't needed as we definitely aren't big enough.
+
+## Who is in a small team?
+
+Fewer than 10 people, but that's the only rule. It may be all stack engineers, there may be a single product person, there could be a customer success person.
+
+## Will this lead to inconsistent design?
+
+Eventually, yes. Other companies have a UX team that build components for everyone to use. Since we currently use Ant Design, we don't need this just yet.
+
+## Can people change teams?
+
+If urgent, although ideally if there is a gap, we'd fill it by hiring and not creating a hole somewhere else. This means people can own each area for longer than a single sprint so they can see the impact of their work longer term as it starts to add up.
```

Comment:
The business moves so quickly, I would suggest a more flexible approach. We did this at TransferWise and it added a lot of value and fresh perspectives to enable e.g. engineers to switch between teams (has to be reasonable of course and not leave big gaps)

Reply:
This makes sense - I think @timgl is a little worried it could constrain people too early. I think the answer may lie in @kpthatsme 's suggestion of grouping slightly differently

Reply:
Fair, and I've since discussed with @piemets - conclusion was that _some_ ability to switch around seems important, but too much means people don't get a sense of ownership.